crypto/ec/ec_key.c

```diff
@@ -820,6 +820,13 @@ void EC_KEY_clear_flags(EC_KEY *key, int flags)
     key->dirty_cnt++;
 }
 
+int EC_KEY_decoded_from_explicit_params(const EC_KEY *key)
+{
+    if (key == NULL || key->group == NULL)
+        return 0;
```

Comment:
Should we return -1 if the arguments are invalid?

Reply:
I am not sure it is worth it as it complicates the callers. But if you insist, I can do it.